jwst/associations/lib/rules_level2b.py

```diff
@@ -760,8 +760,8 @@ def __init__(self, *args, **kwargs):
                                 sources=['numdthpt'],
                             ),
                             DMSAttrConstraint(
-                                name='subpxpns',
-                                sources=['subpxpns'],
+                                name='subpxpts',
+                                sources=['subpxpns', 'subpxpts'],
```

Comment:
Yay for backwards compatibility!

Reply:
and, believe it or not, by design!

Reply:
And what a clever design. One instance here that lists both keywords as sources can then be referenced anywhere else simply using the single constraint name "subpxpts" and it works like magic!